hail/src/main/scala/is/hail/cxx/Emit.scala

```diff
@@ -464,7 +458,7 @@ class Emitter(fb: FunctionBuilder, nSpecialArgs: Int) {
               sb.add(
                 EmitTriplet(f.typ, "",
                   oldPStruct.cxxIsFieldMissing(ov.toString, fieldIdx),
-                  oldPStruct.cxxLoadField(ov.toString, fieldIdx)))
+                  oldPStruct.cxxLoadField(ov.toString, fieldIdx), resultRegion)) //FIXME
```

Comment:
What needs to be fixed?

Reply:
oops, I forgot to remove this once I'd fixed it >.>